### PR TITLE
AP_HAL_Linux: support 16 RCInput channels on Navio2

### PR DESCRIPTION
**Pull request by staroselskii**
This is possible to transmit more than 8 channels over SBUS and now we support that.

### CODE REVIEW

**Comment by lucasdemarchi** [comment]
Failing on Travis but due to other reason apparently. Could you rebase and resubmit?

**Comment by lvale** [comment]
If SBus can handle 16 channels, is there a reason to limit to 14?

**Comment by staroselskii** [comment]
@lvale Thanks for noticing this!

@lucasdemarchi Done.

File: modules/PX4Firmware [hunk 1]
```diff
@@ -1 +1 @@
-Subproject commit a78c598b4ce030dc3556d7a139099b75cf57f692
+Subproject commit 7fd7b2b1b0ecdad69d92e0bf0abcb7849d3826a1
```
Comment (lucasdemarchi) [inline modules/PX4Firmware]: these changes don't belong here.

**Comment by lucasdemarchi** [comment]
I fixed the issue and applied. Thanks.

**Pull request closed without merging**